projects/angular/components/ui-grid/src/ui-grid.component.ts

```diff
@@ -899,7 +909,8 @@ export class UiGridComponent<T extends IGridDataEntry>
 
         this.disableSelectionByEntry = () => null;
         this._fetchStrategy = _gridOptions?.fetchStrategy ?? 'onOpen';
-        this.rowSize = _gridOptions?.rowSize ?? DEFAULT_VIRTUAL_SCROLL_ITEM_SIZE;
+        this.hasHighDensity = Boolean(this._gridOptions?.hasHighDensity);
```

Comment:
nit: try to be consistent when a pattern is available already.
```suggestion
        this.hasHighDensity = this._gridOptions?.hasHighDensity ?? false;
```